src/ToolInfo/ToolInfo.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TwigCsFixer\ToolInfo;
+
+/**
+ * Obtain information about using version of tool.
+ */
+final class ToolInfo implements ToolInfoInterface
+{
+    public const COMPOSER_PACKAGE_NAME = 'k10r/twig-cs-fixer';
+
+    private ?array $composerInstallationDetails = [];
+
+    public function getComposerInstallationDetails(): array
+    {
+        if (null === $this->composerInstallationDetails) {
+            $composerInstalled = json_decode(file_get_contents($this->getComposerInstalledFile()), true);
+
+            $packages = $composerInstalled['packages'] ?? $composerInstalled;
+
+            foreach ($packages as $package) {
+                if (self::COMPOSER_PACKAGE_NAME === $package['name']) {
+                    $this->composerInstallationDetails = $package;
+
+                    break;
+                }
+            }
+        }
```

Comment:
I think there is already something provided by Composer.
Like `InstalledVersions::getVersion()`.

So ToolInfo and ToolInfoInterface shouldn't be needed.